src/main/resources/io/jenkins/plugins/gitlabserver/servers/GitLabServer/config.groovy

```diff
@@ -3,12 +3,13 @@ package io.jenkins.plugins.gitlabserver.servers.GitLabServer
 import io.jenkins.plugins.gitlabserver.servers.GitLabServer
 import lib.FormTagLib
 import lib.CredentialsTagLib
+import org.apache.commons.lang.RandomStringUtils
 
 def f = namespace(FormTagLib)
 def c = namespace(CredentialsTagLib)
 
-f.entry(title: _("Name"), field: "name", "description": "A name for the connection") {
-    f.textbox()
+f.entry(title: _("Name"), field: "name", "description": "A unique name for the connection") {
+    f.textbox(default: String.format("gitlab-%s", RandomStringUtils.randomNumeric(GitLabServer.SHORT_NAME_LENGTH)))
```

Comment:
Feel free to ignore, but I wonder if you should print out the value of SHORT_NAME_LENGTH? The internal name might not mean much to an end user (though to be fair,  `RandomStringUtils.randomNumeric(GitLabServer.SHORT_NAME_LENGTH)))` might not either

Reply:
As of now I am not quite sure as of now that user will never be required to know the server name. If that is the case, in future I shall remove it from user view. In the meantime I will keep a JIRA ticket open for this.

Reply:
I believe this would show up in the UI, right?

Reply:
Yes, it does at this moment. @justinharringa

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-57883

Reply:
I think this is a reasonable approach and a pattern I've seen before in other plugins.